eth/api_debug.go

```diff
@@ -446,3 +447,37 @@ func (api *DebugAPI) GetTrieFlushInterval() (string, error) {
 	}
 	return api.eth.blockchain.GetTrieFlushInterval().String(), nil
 }
+
+func (api *DebugAPI) ExecutionWitness(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) (*stateless.ExecutionWitness, error) {
```

Comment:
We should add a test for this as well. The RPC tests aren't particularly comprehensive but there is typically one for each API that hits on the key functionality.

Reply:
Cool, I'll try to add one that tests re-execution from witness